SegmentEditorLocalThreshold/SegmentEditorLocalThresholdLib/SegmentEditorEffect.py

```diff
@@ -563,3 +567,5 @@ def cropOrientedImage(sourceImageData, roiNode):
 LABEL_VALUE = 1
 SELECTED_ISLAND_VALUE = 2
 OUTSIDE_THRESHOLD_VALUE = 3
+
+ENABLE_SLICE_VIEW_INTERACTION_PARAMETER_NAME = "ENABLE_SLICE_VIEW_INTERACTION"
```

Comment:
Please follow naming conventions
```suggestion
ENABLE_SLICE_VIEW_INTERACTION_PARAMETER_NAME = "SliceViewInteractionEnabled"
```

Reply:
Done